apps/ejabberd/src/mongoose_acc.erl

```diff
@@ -119,59 +123,38 @@ from_map(M) ->
 update(Acc, M) ->
     maps:merge(Acc, M).
 
-%% @doc convert to map so that we can pattern-match on it
--spec to_map(t()) -> map()|{error, cant_convert_to_map}.
-to_map(P) when is_map(P) ->
-    P;
-to_map(_) ->
-    {error, cant_convert_to_map}.
-
--spec put(atom(), any(), t()) -> t().
-put(to_send, Val, Acc) ->
-    % stanza to be sent out may change a few times, and sometimes it carries its own type
-    % (e.g. presence probe), we have to clear previouse value
-    A1 = maps:remove(send_type, Acc),
-    maps:put(to_send, Val, A1);
+-spec put(any(), any(), t()) -> t().
 put(from_jid, Val, Acc) ->
-    ?DEPRECATED,
+    % used only when we have to manually construct an acc (instead of calling from_element)
+    % namely: in c2s terminate, since it is not triggered by stanza, and in some deprecated functions
```

Comment:
According to **Elvis**:
> Line 129 is too long:     % namely: in c2s terminate, since it is not triggered by stanza, and in some deprecated functions.